### PR TITLE
fix discovery after dependency updates

### PR DESCRIPTION
**Pull request by jrhea**

### CODE REVIEW

File: packages/bunnymen-core/src/node.ts [hunk 1]
```diff
@@ -1,246 +1,294 @@
...
+      if (this._peerId != undefined) {
+        this.checkForNewPeers()
+        if (this._currentStep >= frequency) {
+          this.selectLeader()
+          this._currentStep = 0
+        }
+        this._currentStep += this.POLLING_FREQENCY
+      }
+    }, this.POLLING_FREQENCY)
+  }
```
Comment (ryangoree) [inline packages/bunnymen-core/src/node.ts:100]: This is cool, so it polls every 100 ms, but only selects a leader at the frequency from arguments?

File: packages/bunnymen-core/src/node.ts [hunk 2]
```diff
@@ -1,246 +1,294 @@
...
+    var peersLeft = prevPeers.filter(
+      (prevPeer: string) =>
+        prevPeer != this._peerId && !this._peers.includes(prevPeer)
+    )
+    peersLeft.forEach((peer) => this.emit('peerUnsubscribed', peer.toString()))
 
-        const node: IPFS.IPFS = await this._node
-        var prevPeers: string[] = this._peers;
-        this._peers = (await node.pubsub.peers(this._topic)).map(String);
+    var peersJoined = this._peers.filter(
```
Comment (ryangoree) [inline packages/bunnymen-core/src/node.ts:195]: ~~var~~ **const** 🙏

File: packages/bunnymen-core/src/node.ts [hunk 3]
```diff
@@ -1,246 +1,294 @@
...
+          '/ip4/0.0.0.0/tcp/0/ws',
+          '/ip4/0.0.0.0/tcp/0',
+          '/ip4/127.0.0.1/tcp/0/ws',
+          '/ip4/127.0.0.1/tcp/0',
+        ],
+      },
+      connectionManager: {
+        pollInterval: 5000,
+        autoDial: true, // auto dial to peers we find when we have less peers than `connectionManager.minPeers`,
+        minConnections: 20
```
Comment (ryangoree) [inline packages/bunnymen-core/src/node.ts:273]: ~~connectionManager.minPeers~~ **connectionManager.minConnections**

File: packages/bunnymen-core/src/nodeExample.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { Node } from './node.js'
 
 export async function main() {
-    const topic = 'frank'
+    const topic = 'kjkjkjhdgxgcvjdjdmncnxjs'
```
Comment (ryangoree) [inline packages/bunnymen-core/src/nodeExample.ts:4]: My favorite topic. Rolls off the tongue

**Review by ryangoree — APPROVED** [review]
Nice!

File: packages/bunnymen-core/src/node.ts (hunk 1 above)
Comment (jrhea) [reply packages/bunnymen-core/src/node.ts:100]: yessir

File: packages/bunnymen-core/src/node.ts (hunk 2 above)
Comment (jrhea) [reply packages/bunnymen-core/src/node.ts:195]: ill change this in the next PR. this isn't new code though. i think formatters are fighting

**Pull request merged**